### PR TITLE
Jamila development #3

### PR DESCRIPTION
**Pull request by jamila-sabaz**
New pull-request with updated features and removed Navigation bar

### CODE REVIEW

**Comment by jamila-sabaz** [comment]
Lack of comments in the script file - FIXED

**Comment by jamila-sabaz** [comment]
add a blank new line at the end of files (the other html, css, and js files in this PR too) - FIXED

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,18 +1,189 @@
...
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+
+	<head>
```
Comment (lorikeet-s) [inline portfolio/src/main/webapp/index.html]: The indentation seems to have been messed up in this file, could you fix it so you're indenting by 2 spaces again.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -15,14 +15,111 @@
...
+    ['The Good Place', 'Gossip Girl', 'Community', 'Emily in Paris', 'How I met Your Mother'];
+    
+  // Pick a random tv show.
+  const show = shows[Math.floor(Math.random() * shows.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const showContainer = document.getElementById('tv-show-container');
+  //check if the button has been clicked already and if yes collapse container
```
Comment (lorikeet-s) [inline portfolio/src/main/webapp/script.js]: nit: comments should start with a space, then a capital letter and end with a full stop, so they read like a sentence. See comment above for example: // Add it to the page.
Could you fix up the comments in this file.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -15,14 +15,111 @@
 /**
  * Adds a random greeting to the page.
```
Comment (lorikeet-s) [inline portfolio/src/main/webapp/script.js]: This comment seems to have been detached from it's function, and also needs updating.

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -15,14 +15,111 @@
...
+      factHasBeenClicked = false;
+    }
+    //else if not clicked show the container and flag as clicked
+    else{
+      factContainer.innerText = fact;
+      factHasBeenClicked = true;
+  }
+}
+
+//Button for 3 sections about me
```
Comment (lorikeet-s) [inline portfolio/src/main/webapp/script.js]: Typically we use /**/ style comments for function comments, like the other functions in this file.
Can you also update the functions below.

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -1,9 +1,222 @@
+#UTF-8 {
```
Comment (lorikeet-s) [inline portfolio/src/main/webapp/style.css:1]: indentation seems to have been messed up in this file too.

**Review by lorikeet-s — CHANGES_REQUESTED** [review]
Some minor style comments but otherwise looking good, thanks for fixing the PR up for me.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (jamila-sabaz) [reply portfolio/src/main/webapp/index.html]: fixed

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (jamila-sabaz) [reply portfolio/src/main/webapp/script.js]: Fixed.

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (jamila-sabaz) [reply portfolio/src/main/webapp/script.js]: fixed

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (jamila-sabaz) [reply portfolio/src/main/webapp/script.js]: fixed

File: portfolio/src/main/webapp/style.css (hunk 5 above)
Comment (jamila-sabaz) [reply portfolio/src/main/webapp/style.css:1]: fixed

**Comment by jamila-sabaz** [comment]
I fixed everything up but I messed up with branch checkout and accidentally made changes on the new server branch and had to create a new branch with portfolio changes, so I don't lose my precious code. Sorryyyyy it's in a new PR

**Review by lorikeet-s — APPROVED** [review]

**Pull request merged**